filesystem/apps/ansi-term/buffer.rs

```diff
@@ -73,39 +73,39 @@ impl Buffer {
                     self.cursor.y += 1;
                     self.cursor.goto_x(0);
                 }
-            },
+            }
             Action::CursorUp(n) => self.cursor.x = self.cursor.x.saturating_sub(n),
             Action::NextLine(n) => {
                 self.cursor.goto_x(0);
                 self.cursor.y += n;
-            },
+            }
             Action::PreviousLine(n) => {
                 self.cursor.y = self.cursor.y.saturating_sub(n);
                 self.cursor.x = 0;
-            },
+            }
             Action::GotoColumn(x) => self.cursor.goto_x(x),
             Action::Goto(x, y) => self.cursor.goto(x, y),
             Action::GotoStart => self.cursor.goto(0, 0),
             Action::EraseAfter => {
                 self.grid[self.cursor.y].take(self.cursor.x..);
                 self.grid.take(self.cursor.y + 1..);
-            },
+            }
```

Comment:
I don't know. Sometimes it adds them, sometimes it removes them. I can't tell where it does and doesn't do that though, it makes no sense.